includes/definitions/arubaos.yaml

```diff
@@ -5,6 +5,8 @@ icon: aruba
 over:
     - { graph: device_arubacontroller_numaps, text: 'Number of APs' }
     - { graph: device_arubacontroller_numclients, text: 'Number of Clients' }
+mib_dir:
```

Comment:
This isn't actually needed. The os name is used as a mib dir by default if it exists.